e-commerce/payment-backend/dbos-config.yaml

```diff
@@ -9,13 +9,13 @@ database:
   username: 'postgres'
   password: ${PGPASSWORD}
   app_db_name: 'payment'
-  user_database: 'payment' # Will be deprecated, use app_db_name instead
   connectionTimeoutMillis: 3000
   app_db_client: 'knex'
-  user_dbclient: 'knex' # Will be deprecated, use app_db_client instead
   migrate: ['npx knex migrate:latest']
   rollback: ['npx knex migrate:rollback']
 runtimeConfig:
   port: 8086
 application:
-  frontend_host: 'http://localhost:8000'
+  frontend_host: 'http://localhost:8086'
+  # For DBOS Cloud:
+  #frontend_host: 'https://<username>-<app-name>.cloud.dbos.dev'
```

Comment:
Should  this not be https://<username>-<app-name>.<clustername>.cloud.dbos.dev
May be not -- I guess there is no clustername for prod
So might be ok

Reply:
One thing I could do is replace <app-name> with the name from package.json... that's unlikely to get changed. But it does sort of show the formula for the URL.

Reply:
there is no cluster name for prod

Reply:
It should be `https://<username>-payment-backend.cloud.dbos.dev`